README.md

```diff
@@ -5,6 +5,7 @@ A declarative network manager API for hosts.
 [![Coverage Status](https://coveralls.io/repos/github/nmstate/nmstate/badge.svg?branch=master)](https://coveralls.io/github/nmstate/nmstate?branch=master)
 [![PyPI version](https://badge.fury.io/py/nmstate.svg)](https://badge.fury.io/py/nmstate)
 [![Fedora Rawhide version](https://img.shields.io/badge/dynamic/json.svg?label=Fedora%20Rawhide&url=https%3A%2F%2Fapps.fedoraproject.org%2Fmdapi%2Frawhide%2Fpkg%2Fnmstate&query=%24.version&colorB=blue)](https://apps.fedoraproject.org/packages/nmstate)
+[![Code Style](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/python/black)
```

Comment:
This can probably be merged without much discussion. However something should say that we do not follow it completely, so it should be a little bit more grey ;-)

Reply:
We do follow it completely. Black customization is minimal and existing options are fully supported.
We enforce non default line length and do not enforce quotation rules, both valid under Black.
(If we would have patched Black to support single quotation, then it would have merit a remark).

Regarding documentation, I think it is stated in the tox.ini file and the commit it introduced it. We can even mention it in the changelog for the next version.